gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -1098,6 +1105,19 @@ private ContextAwareMeter getGroupMeterForDag(String dagId, String meterName, Ma
           group -> metricContext.contextAwareMeter(MetricRegistry.name(ServiceMetricNames.GOBBLIN_SERVICE_PREFIX, group, meterName)));
     }
 
+    /**
+     * Used to track metrics for different specExecutors to detect issues with the specExecutor itself
+     * @param dagNode
+     * @param meterName
+     * @param meterMap
+     * @return
+     */
+    private ContextAwareMeter getExecutorMeterForDag(DagNode<JobExecutionPlan> dagNode, String meterName, Map<String, ContextAwareMeter> meterMap) {
+      String executorName = dagNode.getValue().getSpecExecutor().getUri().toString();
```

Comment:
if we lack a `DagManagerUtils.getExecutorName`, should we add?

Reply:
I can add that, that's a good point :)